routes/web.php

```diff
@@ -54,6 +57,11 @@
 Route::get('demo', DemoController::class);
 Route::post('process-demo', ProcessDemoFormController::class);
 
+Route::get('custom-rule/{uuid}', CustomRuleDetailController::class)
+    ->whereUuid('uuid');
+Route::get('custom-rule', CustomRuleController::class);
+Route::post('process-custom-rule', ProcessCustomRuleFormController::class);
```

Comment:
this need to be tested locally first for "dev" only try, I checked locally, using this seems work:

```php
if ((int) env('APP_DEBUG') === 1) {
    // ...
}
```

per `.env.prod` for server and `.env.dist` for local

https://github.com/rectorphp/getrector-com/blob/6550ad66a3ff621b211208767ef85f9ab3cdfdd7/.env.prod#L2

https://github.com/rectorphp/getrector-com/blob/6550ad66a3ff621b211208767ef85f9ab3cdfdd7/.env.dist#L2

not sure why `APP_ENV` show as "local"